recipe/recipe.yaml

```diff
@@ -52,7 +53,15 @@ outputs:
           imports:
             - rattler
           python_version: ["${{ python_min ~ '.*' }}", "3.12.*"]
-
+      - script:
+          - if: unix and is_abi3
+            then: abi3audit ${{ SP_DIR }}/spam.abi3.so -s -v --assume-minimum-abi3 ${{ python_min }}
```

Comment:
@baszalmstra @wolfv is SP_DIR not defined?

Reply:
Reference: https://github.com/prefix-dev/rattler-build/blob/26e22fc8bf1b9145b6a63743e815d7d5ea65ad83/src/env_vars.rs#L41-L42

Reply:
SITE_PACKAGES_DIR=$(python3 -c "import site; print(site.getsitepackages()[0])")

Something like this should do the trick in the test I suppose

Reply:
What about windows? Do you think we shouldn't define this for test execution time?